runtime/doc/treesitter.txt

```diff
@@ -1206,4 +1223,10 @@ LanguageTree:trees()                                    *LanguageTree:trees()*
     Returns all trees this language tree contains. Does not include child
     languages.
 
+==============================================================================
+QUERY EDITOR HIGHLIGHT GROUPS                          query-highlight-groups
+
+                                                  *hl-TsQueryEditorCapture*
```

Comment:
I'm not sure if this is the right place to document this highlight group. I don't want it to be confusing with the other highlighting section...

Reply:
I'd use the same logic as for the tree view: just use one of the default search highlights (`CurSearch`?) No need for a separate highlight (for now).

Reply:
I think `NonText` is more appropriate, since it's a virtual text thing and `CurSearch` is usually a very bright highlight.

Reply:
Oh, sorry, this is for the capture name virtual text, not the highlight; I didn't read properly. Then I'd use the same one that the tree view uses for the language names (if you press `I`)?

Reply:
Oh that's `Title`. I'll change it then.